pegasus/sites.v3/code.org/layouts/wide.haml

```diff
@@ -18,12 +18,9 @@
   .push
 
 -if @header['theme'] == 'responsive_wide'
-  -if page_translated? && !partner_site?
```

Comment:
partner_site refers to chapter partner sites, like in this old list: https://github.com/code-dot-org/code-dot-org/commit/e2dd11a62cceb32628b3583f061dfd9bbe0434eb
We have deprecated all the partner sites: https://github.com/code-dot-org/code-dot-org/pull/22865